types/create-hash/index.d.ts

```diff
@@ -1,16 +1,22 @@
 // Type definitions for create-hash 1.2
 // Project: https://github.com/crypto-browserify/createHash
 // Definitions by: BendingBender <https://github.com/BendingBender>,
-//                 Konstantin Yuriev <https://github.com/gallowsmaker>
-//                 Max Boguslavskiy <https://github.com/maxbogus>
+//                 Konstantin Yuriev <https://github.com/gallowsmaker>,
+//                 Max Boguslavskiy <https://github.com/maxbogus>,
+//                 Jordan Sexton <https://github.com/jordansexton>,
```

Comment:
I think you should remove comma at the end of the sentence.
Commit changes and run "npm test" again.

Reply:
Got it, I'll try that. Thanks!